src/css/custom.css

```diff
@@ -93,6 +95,54 @@
   font-display: swap;
 }
 
+@font-face {
```

Comment:
@deekshithepns this needs to be only two fonts

Reply:
Initially able to add only 2 fonts base and monospace. but, there is a requirement for 3 kind of fonts, what to do for that. without adding that font in infima variables it will not get rendered.

Reply:
I think I added fonts in the same way

Reply:
ok